0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,248 @@
+Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen ledgers and direct peers.
+
+## Cyclic Transaction
+A transaction where the destination account is the same account, on the same ledger, as the source account. This can be useful
+when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage
+of changing exchange rates).
+
+# Destination Account
+The account of the receiver, whose address is included in the
+[interledger packet](#interledger-packet).
+Note that anyone can claim to have a certain [Interledger address](#interledger-address); address ownership is not enforced.
+
+# Destination Amount
+The amount to be received by the receiver.
+
+# Destination Ledger
+The ledger on which the final receiver of an Interledger payment holds their account
+
+## Distributed Ledger
+A ledger that is operated by a group of entities and/or run on multiple servers. This term is used somewhat interchangeably with "Blockchain" or "Decentralized Ledger".
+
+## Exchange Rate
+The price of one ledger's asset in terms of another ledger's asset.
+Connectors may generate revenue from the difference in value between incoming and outgoing transfers.
+Senders may request quotes from multiple connectors to determine the best price before sending a payment.
+The exchange rate between source and destination is determined by the product of exchange rates at each hop.
+
+## Executed
+A final state of a transfer indicating that the funds were transferred to the account of the immediate recipient.
+
+## Execution Phase
+The second phase of an ILP payment, in which the fulfillment is passed back from the receiver, via the connector(s), to the sender.
+
+## Final State
+The state an interledger payment ends up in. Has to be either
+[rejected](#rejected) or [executed](#executed).
+
+## Fulfillment
+See [conditional transfer](#conditional-transfer).
+
+## Hold
+The state of part of the sender's, connector's, and/or receiver's account balance, during which that balance is not readily available, in relation to
+a specific ILP payment. This state starts after an ILP payment is prepared, and ends when the payment is either executed or rejected.
+
+## Hop
+One of the transfers that make up a payment. Specifically, a 'multi-hop' payment is one involving more than one ledger, and thus at least one connector.
+Likewise, 'best hop', as used in the ilp-connector source code when making routing decisions, is best interpreted as 'best next transfer'.
+
+## IL-RFC
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in
+[gh:interledger/rfcs](https://github.com/interledgers/rfcs).
+
+## Incoming (Transfer, Ledger, Amount)
```

Comment:
Wouldn't you also describe the destination transfer as an incoming transfer from the perspective of the receiver? I don't think this term has meaning in general, it's relative to the party you're talking about